triphouse-project/styles/advantages-styles.css

```diff
@@ -26,27 +26,27 @@ body{
     font-style: normal;
 }
 
+.wrapper{
+    margin-left: auto;
+    margin-right: auto;
+    box-sizing: border-box;
+}
+
 .offer-panel{
-    width: 1440px;
-    height: 569px;
     background: #FFFFFF;
     box-sizing: border-box;
```

Comment:
you can specify box-sizing: border-box for all elements by *{  box-sizing: border-box } to avoiding dublicating of styles